### ISSUE TITLE
Creative

### ISSUE THREAD

**Issue by PeterkalM** [tomudding/Player-Visibility#12]
I will open this creative issue again. Because doesn't InventoryCreativeEvent cancel it? At least from my testing InventoryCreativeEvent does cancel middle click, but just sometimes doesn't display it right.

**Comment by PeterkalM** [comment]
For example this blocks middle click for anvils:
```
    public void creative(InventoryCreativeEvent event) {
        if (event.getCursor().getType() == Material.ANVIL) {
            event.setCancelled(true);
        }
    }
```

**Comment by tomudding** [comment]
I tested the `InventoryCreativeEvent` for more than an hour and I couldn't get it to work, I'll look into it again because I think I might have made a mistake in the code I submitted.

**Comment by PeterkalM** [comment]
I looked in to the code and I think ItemStack amount messes some event checks up. Insted of `equals(item)` you should try `isSimilar(item)`.

**Comment by tomudding** [comment]
I have changed that in a new test build (which I am testing today, tests are positive). I'll try to push and upload v2.5.2 this weekend (I don't have the time to do it tomorrow).

**Comment by tomudding** [comment]
Fixed with #13.
